requirements.txt

```diff
@@ -2,19 +2,18 @@ aiohttp==3.9.1
 aiohttp-retry==2.8.3
 aiosignal==1.3.1
 alembic==1.12.0
-alembic-utils==0.8.1
+alembic-utils==0.8.2
 amqp==5.1.1
 asn1crypto==1.5.1
 async-timeout==4.0.3
 attrs==23.1.0
 Authlib==1.2.1
-backports.zoneinfo==0.2.1
 bcrypt==4.0.1
 billiard==4.1.0
 bleach==6.0.0
 blinker==1.6.2
-boto3==1.28.32
-botocore==1.31.32
+boto3==1.34.7
+botocore==1.34.7
```

Comment:
These are top level dependencies that should also be updated in requirements-app.txt.

Reply:
Done.